src/client/components/configuration/tree/tree.tsx

```diff
@@ -0,0 +1,68 @@
+import * as classnames from 'classnames'
+import { observer } from 'mobx-react'
+import * as React from 'react'
+
+import FileIcon from './file.svg'
+import FolderIconOpen from './folder-open.svg'
+import FolderIcon from './folder.svg'
+import * as style from './style.css'
+
+export interface Node {
+  label: string,
+  expanded: boolean,
+  leaf: boolean,
+  selected: boolean,
+  children?: Node[]
+}
```

Comment:
You can remove all the commas from these interface definitions, e.g.

```
export interface Node {
  label: string
  expanded: boolean
  leaf: boolean
  selected: boolean
  children?: Node[]
}
```

I'd put it in a tslint rule if there was one (https://github.com/palantir/tslint/issues/960)...